runtime/drivers/blob/blobdownloader.go

```diff
@@ -82,6 +92,10 @@ func (opts *Options) validate() {
 	if opts.GlobPageSize == 0 {
 		opts.GlobPageSize = 1000
 	}
+	if opts.BatchSizeBytes == 0 {
+		// 5 GB
+		opts.BatchSizeBytes = 5 * 1024 * 1024 * 1024
```

Comment:
given that we are defaulting to 2 slots, how about reducing this to 2G ?

Reply:
Yeah I also think we should do some benchmarks for this and set it slightly defensively.

Reply:
Seems like a good idea

Reply:
Done. Changed to 2G.